rdr_service/genomic/genomic_job_components.py

```diff
@@ -308,11 +312,11 @@ def _ingest_gc_manifest(self, data, _site):
                         # RDR may need to do something with them in the future
 
                     else:
-                        logging.error(f'Invalid collection tube ID: {collection_tube_id}'
-                                        f' or genome_type: {genome_type}')
-
-                        # Aborting the job if invalid collection tube ID found.
-                        return GenomicSubProcessResult.ERROR
+                        #return GenomicSubProcessResult.ERROR
+                        logging.error(f"Missing collection tube ID: {collection_tube_id}, "
+                                      f"biobank id: {row_copy['biobankidsampleid'].split('_')[0]}, "
```

Comment:
Is anything lost for debugging when dropping the `sampleid` part of the `biobankidsampleid` value from the logged error?

Reply:
For this particular log message, only the `biobank_id` and `collection_tube_id` are needed. The sample ID is not important here. We are getting the sample ID from the Biobank, but the lookup is failing because of `collection_tube_id` and `biobank_id`.